CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 4.0.2
+
+* Updated README instructions to Swift 3.
```

Comment:
Sorry to be nitpicking, but could you please add 2 spaces at the end of that line?

That's a trick related to Markdown formatting: adding two spaces at the end of a line allows to go to the next line while staying in the same paragraph/bullet point